IntegrationTests/lib.js

```diff
@@ -6,6 +6,8 @@ const promisify = require("util").promisify;
 const fs = require("fs");
 const readFile = promisify(fs.readFile);
 
+Error.stackTraceLimit = Infinity;
```

Comment:
Collecting all call stacks has some performance overheads, so it should be disabled when benchmarking